PWGJE/Tasks/jetfinderhfQA.cxx

```diff
@@ -64,8 +63,24 @@ struct JetFinderHFQATask {
   std::vector<bool> filledJetR;
   std::vector<double> jetRadiiValues;
 
+  Service<o2::framework::O2DatabasePDG> pdgDatabase;
+  int candPDG;
+  double candMass;
+
   void init(o2::framework::InitContext&)
   {
+
+    if constexpr (std::is_same_v<std::decay_t<CandidateTableData>, soa::Join<aod::HfCand2Prong, aod::HfSelD0>>) { // Note : need to be careful if configurable workflow options are added later
+      candPDG = static_cast<int>(pdg::Code::kD0);
+    }
+    if constexpr (std::is_same_v<std::decay_t<CandidateTableData>, soa::Join<aod::HfCand3Prong, aod::HfSelLc>>) {
+      candPDG = static_cast<int>(pdg::Code::kLambdaCPlus);
+    }
+    if constexpr (std::is_same_v<std::decay_t<CandidateTableData>, soa::Join<aod::HfCandBplus, aod::HfSelBplusToD0Pi>>) {
+      candPDG = static_cast<int>(pdg::Code::kBPlus);
+    }
+    candMass = pdgDatabase->Mass(candPDG);
```

Comment:
You can directly assign `candMass` value from `PDG.h` without the need of `pdgDatabase` and `candPDG`.